float64 timestamp
float64 ch1_mv
float64 ch1_ua
float64 ch2_mv
float64 ch2_ua
float64 pm_mv